src/js/panels.js

```diff
@@ -284,21 +284,21 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
      * Text to speech panel
      */
     fluid.defaults("gpii.firstDiscovery.panel.speakText", {
-        gradeNames: ["fluid.prefs.panel", "autoInit"],
+        gradeNames: ["fluid.prefs.panel", "gpii.firstDiscovery.attachTooltip.renderer", "autoInit"],
         preferenceMap: {
             "gpii.firstDiscovery.speak": {
                 "model.speak": "default"
             }
         },
-        modelRelay: {
-            source: "{that}.model",
+        modelRelay: [{
+            source: "{that}.model.speakChoice",
             target: "{that}.model.speak",
             // Setup the backward restriction to prevent the component instantiation writes back to
-            // the central model that ends up wiping out the saved prefs at the page reload.
-            backward: "liveOnly",
+            // the central model that results in wiping out the saved prefs at the page reload.
+            forward: "liveOnly",
             singleTransform: {
                 type: "fluid.transforms.valueMapper",
-                inputPath: "speakChoice",
+                inputPath: "",
```

Comment:
I tried not specifying inputPath, would break the transformation.

The reason I changed `source: "{that}.model"` to `source: "{that}.model.speakChoice"` is the former looks like any model change on the entire model would trigger the model relay while the only piece this relay cares about is changes on "speakChoice".